arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -165,5 +165,52 @@ public async Task GetVertexCollectionsAsync_ShouldThrow_WhenGraphDoesNotExist()
             Assert.Equal(HttpStatusCode.NotFound, apiError.Code);
             Assert.Equal(1924, apiError.ErrorNum);
         }
+
+        [Fact]
+        public async Task PostGraphAsync_ShouldSucceed()
+        {
+            var graphName = nameof(PostGraphAsync_ShouldSucceed) + "_clx";
+            var response = await _client.PostGraphAsync(new PostGraphBody
+            {
+                Name = graphName,
+                EdgeDefinitions = new List<EdgeDefinition>
+                {
+                    new EdgeDefinition
+                    {
+                        From = new string[] { "fromclx" },
+                        To = new string[] { "toclx" },
+                        Collection = "clx"
+                    }
+                }
+            });
+
+            Assert.Equal(HttpStatusCode.Accepted, response.Code);
+            Assert.Single(response.Graph.EdgeDefinitions);
+            Assert.Equal(graphName, response.Graph.Name);
+        }
+
+        [Fact]
+        public async Task PostGraphAsync_ShouldThrow_WhenInvalidName()
```

Comment:
What about `WhenGraphNameIsInvalid` instead? Just to make it more explicit. In future we might add some tests that are related to edge or vertex collection names.